evm/src/modules/TokenGateway.sol

```diff
@@ -112,11 +112,10 @@ contract TokenGateway is BaseIsmpModule {
     mapping(bytes32 => address) private _erc6160s;
     // mapping of token identifier to erc20 contracts
     mapping(bytes32 => address) private _erc20s;
-    // foreign to local asset identifier mapping
-    mapping(bytes32 => bytes32) private _assets;
 
     // User has received some assets, source chain & nonce
     event AssetReceived(bytes source, uint256 nonce);
+    event Teleport(address from, bytes dest, uint256 amount, uint256 fee, uint64 timeout);
```

Comment:
relayer fee & time out is captured in `PostRequestEvent`, lets remove. Let's instead add the request commitment. 
```suggestion
    event Teleport(address from, address to, uint256 amount, bool redeem, bytes32 requestCommitment);
```

We'll probably need to modify `IDispatcher` for this

Reply:
We are already emitting commitment here:
https://github.com/polytope-labs/hyperbridge/blob/21c42e09d24eb2da20c6d2653bd7916c304d6bbc/evm/src/hosts/EvmHost.sol#L444

Is there really a need to emit requestCommitment again and also going through hashing the request to get it?